app/main/plugins.js

```diff
@@ -5,28 +5,53 @@ const plist = require('plist');
 const deepAssign = require('deep-assign');
 const is = require('is_js');
 const MarkdownIt = require('markdown-it');
-const { PLUGIN_PATH } = require('../../utils/paths');
+const {
+  CORE_PLUGIN_PATH,
+  DEXT_PATH,
+} = require('../../utils/paths');
 const { MAX_RESULTS } = require('../constants');
 
 /**
+ * Attempt to read the config.json file in the DEXT_PATH
+ * @return {Promise}
+ */
+const readConfig = () => (new Promise((resolve, reject) => {
+  const file = `${DEXT_PATH}/config.json`;
+  fs.readFile(file, (err, data = '{}') => {
+    if (err) {
+      reject(err);
+      return;
+    }
+    resolve(JSON.parse(data));
+  });
+}));
```

Comment:
Yes, this function should be moved to `dext-core-utils` since we'll most likely be reusing it on `dext-cli`.